Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -362,6 +362,56 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 }
 /*endregion*/
 
+/*region bocListValidationRow*/
+div.bocList div.bocListTableBlock tr.bocListValidationRow
+{
+  border-bottom: var(--boclist-tabledata-border);
+  border-top: var(--boclist-tabledata-border);
+  border-top-style: dashed;
+  color: red;
+}
+
+div.bocList table.bocListTable tr.bocListValidationRow.selected td
+{
+  background-color: var(--color-highlight);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow td ul
+{
+  margin: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow div
+{
+  width: 300px;
```

Comment:
We shouldn't use PX. instead, we can go with EM and a reasonable number of characters that should be used, e.g. 20?